### PR TITLE
[CAMEL-13704] - Added PR template file with instructions on what to include

### PR DESCRIPTION
**Pull request by Xxyumi-hub**
CAMEL-13704 - https://issues.apache.org/jira/projects/CAMEL/issues/CAMEL-13704?filter=allopenissues
I have created a PR template file for this repo and plan to add the same template to the camel-quarkus repo

### CODE REVIEW

File: PR-template.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
+Below are the contribution guidelines:
+https://github.com/apache/camel/blob/master/CONTRIBUTING.md
```
Comment (nicolaferraro) [inline PR-template.txt]: They are here https://camel.apache.org/camel-k/latest/developers.html

File: PR-template.txt [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
+[ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
+[ ] Run `mvn clean install -Psourcecheck` in your module with source check enabled to make sure basic checks pass and there are no checkstyle violations. A more thorough check will be performed on your pull request automatically.
```
Comment (nicolaferraro) [inline PR-template.txt]: Does not apply

File: PR-template.txt [hunk 3]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
+[ ] Each commit in the pull request should have a meaningful subject line and body.
+[ ] If you're unsure, you can format the pull request title like `[CAMEL-XXX] Fixes bug in camel-file component`, where you replace `CAMEL-XXX` with the appropriate JIRA issue.
```
Comment (nicolaferraro) [inline PR-template.txt]: Fix #issuenr is the right format

File: PR-template.txt [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[ ] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/CAMEL) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
```
Comment (nicolaferraro) [inline PR-template.txt]: We don't use Jira for this repo, we use github issues

File: PR-template.txt (hunk 1 above)
Comment (Xxyumi-hub) [reply PR-template.txt]: Thank you for the feedback! How does the below-proposed changes look?

[ ] Make sure there is a [Github issue](https://github.com/apache/camel-k/issues) filed for the change (before you start working on an issue).  One issue should ideally be fixed by not more than one commit and the other way round, each commit should fix just one issue, without pulling in other changes.
[ ] The pull request description should explain what the pull request does, how, and why. If the info is available in the associated issue or some other external document, a link is enough.
[ ] Phrases like Fix #<issueNumber> or Fixes #<issueNumber> will auto-close the named issue upon merging the pull request. Using them is typically a good idea.
[ ] Contributor guide is your good friend: https://camel.apache.org/camel-k/latest/developers.html

**Comment by oscerd** [comment]
> Looks good to me but this should go under .github directory and renamed

I mean this should be done here:
https://github.com/apache/camel-k/blob/master/.github/pull_request_template.md

**Comment by Xxyumi-hub** [comment]
Updated!! Thank you for the detailed feedback

**Review by oscerd — APPROVED** [review]
Not sure if this is still needed, but if it is needed we should merge

**Pull request closed without merging**